public/images/magnifying-glass.svg

```diff
@@ -2,6 +2,23 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <svg width="14px" height="14px" viewBox="488 384 14 14" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-  <path d="M495.5,391.5 L500.200877,396.200877" id="Line" stroke="#4A90E2" stroke-width="1.25" stroke-linecap="round" stroke-linejoin="round" fill="none"></path>
-  <ellipse id="Oval-6" stroke="#4A90E2" stroke-width="1.25" fill="#FFFFFF" fill-rule="evenodd" cx="493.5" cy="389.5" rx="4.5" ry="4.5"></ellipse>
+  <defs>
+    <style>
+      use:not(:target) {
+        display: none;
+      }
+      #blue {
+        stroke: #4A90E2;
+      }
+      #grey {
+        stroke: #dde1e4;
+      }
```

Comment:
We're getting away from embedding CSS in SVGs. That's why we go through great lengths to embed them right in HTML using the loader stuff. Can you move this CSS into the component's CSS that you are using it in? It makes it much easier to reuse SVGs.